apptools/logger/plugin/tests/test_logger_service.py

```diff
@@ -0,0 +1,43 @@
+from email.mime.multipart import MIMEMultipart
+import unittest
+
+try:
+    # On Python 3, mock is part of the standard library,
+    from unittest import mock
+except ImportError:
+    # Whereas on Python 2 it is not.
+    import mock
+
+from apptools.logger.plugin.logger_service import LoggerService
+
+
+class LoggerServiceTestCase(unittest.TestCase):
+    def test_create_email_message(self):
+        logger_service = LoggerService()
+        with mock.patch.object(logger_service, 'whole_log_text')\
+                as mocked_log_txt:
+            mocked_log_txt.return_value = "Dummy log data"
+            msg = logger_service.create_email_message(
+                fromaddr='', toaddrs='', ccaddrs='', subject='', priority=''
+            )
+        self.assertIsNotNone(msg)
```

Comment:
I'd drop this, unless there's some a priori reason to expect the msg to be `None`. This check is redundant with the `assertIsInstance` check below.